ftplugin/python/SimpylFold.vim

```diff
@@ -6,7 +6,7 @@ let b:loaded_SimpylFold = 1
 let s:blank_regex = '\v^\s*(\#.*)?$'
 let s:def_regex = '^\%(\s*\%(class\|def\) \w\+\|if\s*__name__\s*==\s*''__main__'':\s*\)'
 let s:multiline_def_end_regex = '):$'
-let s:docstring_start_regex = '^\s*\("""\|''''''\)\%(.*\1\s*$\)\@!'
+let s:docstring_start_regex = '^\s*\("""\|r"""\|''''''\)\%(.*\1\s*$\)\@!'
```

Comment:
Raw strings can be triply single-quoted as well (`r'''...'''`). This pattern only matches the double-quoted variant. Additionally, it is legal to use a capital `R` instead of lowercase `r`. Something like `[rR]?` should address these cases.